dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -205,7 +205,7 @@ public Workflow setupWorkflowVersions(String repositoryId, Workflow workflow, Op
                     String content = extractGitHubContents(descriptorContents);
 
                     // TODO: Is this the best way to determine file type? I don't think so
-                    // Should be workflow.getDescriptorType().equals("cwl") - though enum is better!
+                    // Should be workflow.getDescriptor_type().equals("cwl") - though enum is better!
```

Comment:
Let's not use "_" in Java code actually, don't think this will pass the current style either or at least I hope it doesn't